.travis.yml

```diff
@@ -0,0 +1,28 @@
+sudo: false
+
+language: node_js
+
+node_js:
+  - 4
+  - 6
+
+cache:
+  directories:
+    - node_modules
+
+before_install:
+  - npm config set spin false
+
+install:
+  - npm install
+  - npm install -g codecov
+  # - npm install -g bithound
```

Comment:
Uncomment lines with `bithound`, and their service will lint and check your code too (additionally to eslint)